source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModel/AdaptiveCardsSharedModel.vcxproj

```diff
@@ -121,6 +121,7 @@
     <ClCompile Include="..\..\ObjectModel\Column.cpp" />
     <ClCompile Include="..\..\ObjectModel\ColumnSet.cpp" />
     <ClCompile Include="..\..\ObjectModel\Container.cpp" />
+    <ClCompile Include="..\..\ObjectModel\CustomObjectDelegateForIOS.cpp" />
```

Comment:
><ClCompile Include="..\..\ObjectModel\CustomObjectDelegateForIOS.cpp" /> [](start = 4, length = 72)

Is this a merge error or something? This PR doesn't add this file.

Reply:
thanks, it must be.

---
In reply to: [172385623](https://github.com/Microsoft/AdaptiveCards/pull/1210#discussion_r172385623) [](ancestors = 172385623)